doc-content/enterprise-only/springboot/bus-app-import_proc.adoc

```diff
@@ -0,0 +1,33 @@
+[id='bus-apps-inport_{context}']
+= Importing business applications projects into and deploying from {CENTRAL} 
+
+You can import a business assets project that is part of a {PRODUCT} business application into {CENTRAL} and then deploy that project to a running server instance.
```

Comment:
instead of running server instance let's call it running service instance (it should refer to business application service project running)